ipmitool/ipmitool.spec

```diff
@@ -1,14 +1,14 @@
 Name:         ipmitool-xcat
 Summary:      ipmitool - Utility for IPMI control
-Version:      1.8.15
-Release:      3
+Version:      1.8.17
+Release:      0
 License:      BSD
 Group:        Utilities
 Packager:     IBM Corp.
 Source:       ipmitool-%{version}.tar.gz
 Patch:        ipmitool-%{version}-saneretry.patch
 Patch2:       ipmitool-%{version}-rflash.patch
-Patch3:       ipmitool-%{version}-solactivate.patch
+#Patch3:       ipmitool-%{version}-solactivate.patch
```

Comment:
@pdlun92  I thought we had taken a look at this solactivate.patch and I thought we saw that it was included in 1.8.17 ,but now i'm looking at the source code and I don't see the following patch: 

https://github.com/xcat2/xcat-dep/blob/master/ipmitool/ipmitool-1.8.15-solactivate.patch

I don't see it in this file:  https://sourceforge.net/p/ipmitool/source/ci/IPMITOOL_1_8_17/tree/lib/ipmi_sol.c  

I think we need to create this new patch  `ipmitool-1.8.17-solactivate.patch` and rebuild the rpms.  :-( 

This is the thread that talks about it, but i thought I had seen a "target" for when this patch would be integrated,  I can't find it at this time.  https://sourceforge.net/p/ipmitool/mailman/message/34370504/